### ISSUE TITLE
Quick action button misplaced

### ISSUE THREAD

**Issue by kadlecf** [osmandapp/OsmAnd#18737]
### Description

On my map view screen, I have been using the Quick action button. Since a recent update, the button is placed quite far away from the  map border, which disturbs the map view. This occurs in the vertical view layout; when the screen is rotated to the landscape mode, its position, as it seems to me, is correct, close to the "current position" button as it used to be before.

### Steps to reproduce

Start OsmAnd+, activate the Quick action button

### Actual result

The Quick action button appears in the middle of the screen
![Screenshot_20231214-123525](https://github.com/osmandapp/OsmAnd/assets/81177285/b3da88a0-43f1-45bc-94f2-8edd162b5cd5)

### Expected result

I would expect the button to be placed at the bottom of the screen.

![Screenshot_20231214-123617](https://github.com/osmandapp/OsmAnd/assets/81177285/4a7a0d13-b18e-428b-9620-8bc8eac34932)

### Your Environment (required)

**WARNING** Crash-Logs **MAY** contain information you deem sensitive.
Review this **CAREFULLY** before posting your issue!

```
OsmAnd Version: 4.5.10.
Android/iOS version: Android 12, kernel 4.14.190-perf+
Device model: Nokia 5.3
Crash-Logs: ?
```

**Comment by pebogufi** [comment]
Just make along tap on it, then move it to the position, you prefer.

**Comment by yuriiurshuliak** [comment]
The bug has been reproduced. Follow these steps to recreate it:

-Activate the Quick action button.
-Place the quick action for portrait mode at the upper third of the screen.
-Experiment with flipping the phone across various modes.

For a temporary fix, adjust the button's position in portrait mode to another spot, as demonstrated in the video.

OsmAnd TestFlight 4.6.2.6


https://github.com/osmandapp/OsmAnd/assets/127092082/23209caf-104a-49d0-8d3f-8b9b37c4407a

**Comment by DmitryAlexei** [comment]
I would consider this behavior on iOS not as a bug, although as a feature. With that option, the user can set two different positions for the quick action button - one for portrait mode, one for landscape mode.
Anyway, this issue should be tested on **Android** device. 
In my case (latest nightly build), OsmAnd behaves the same, as on iOS: user can set different positions for different modes.

**Comment by kadlecf** [comment]
I agree that once it is possible to move the button to the border, it is no more annoying, so it should not be considered as a bug. I think just that this possibility to move it should be documented somewhere.
